test/Rings/nmod-localizations.jl

```diff
@@ -0,0 +1,164 @@
+#################################################################
+# Localizations of residue rings ℤ/nℤ                           #
+#################################################################
```

Comment:
Same as above: this code should be in an example file, and only the `@testset`s should remain in this file, if at all?

Reply:
Personally, I don't have any strong opinions on where the code should be. I would just like to keep it somewhere where people can find it and see how things are done.